src/sentry/api/endpoints/organization_events_trace.py

```diff
@@ -96,30 +162,15 @@ def query_trace_data(trace_id, params):
     ]
 
 
-class OrganizationEventsTraceEndpointBase(OrganizationEventsV2EndpointBase):
-    def has_feature(self, organization, request):
+# TODO(wmak): remove ignore once the base is typed
+class OrganizationEventsTraceEndpointBase(OrganizationEventsV2EndpointBase):  # type: ignore
+    def has_feature(self, organization: Organization, request: HttpRequest) -> Any:
```

Comment:
This should return a bool right? If `features.has` returns any you should do `assert isinstance(x, bool)` or `bool(x)`.